utils/create-keying-sheet-tfm-facilities/helpers/get-portal-user-id.ts

```diff
@@ -0,0 +1,12 @@
+import { PortalUser } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+import { mongoDbClient } from '../database-client';
+
+export const getPortalUserIdOrFail = async (): Promise<ObjectId> => {
```

Comment:
Where is this used? Wonder if we need to be careful with what type of user we are attributing to the deal (and if someone with a wrong role attached to an action might cause errors later?) let me know if I'm overthinking this

Reply:
It's actually only used in the audit records, so is perhaps actually overkill anyway since I'm not sure we parse them? But basically I replaced any ids that were pulled as object ids with real object ids. There are other ids stored as just string which I have not made dynamically accurate e.g. deal has a nested field inside `portalActivities` for the checker on one of the objects in there. Just opening the deal and looking didn't throw any errors. I am hoping if someone wanted to test something about a deal they would not continue a journey from this deal however, and instead would ideally use a fresh one.